test/controllers/thesis_controller_test.rb

```diff
@@ -461,14 +461,6 @@ class ThesisControllerTest < ActionDispatch::IntegrationTest
     assert theses.second.grad_date <= theses.third.grad_date
   end
 
-  test 'can be sorted by name' do
-    sign_in users(:admin)
-    get process_path(sort: 'name')
-    theses = @controller.instance_variable_get(:@theses)
-    assert theses.first.user.surname <= theses.second.user.surname
-    assert theses.second.user.surname <= theses.third.user.surname
-  end
```

Comment:
I suspect that we've removed this test because the sort order is problematic now with the link table in place - which would make sense - but it looks like the UI still includes the buttons for resorting this display - and the Name button now causes an application error: https://thesis-dropbox-pr-596.herokuapp.com/process?status=any

Could that button be removed (and actually, probably the entire "Sort by" control can be removed, since Date is the only remaining option)